lib/src/hook_file.dart

```diff
@@ -0,0 +1,69 @@
+import 'package:bdd_widget_test/src/generator_options.dart';
+import 'package:bdd_widget_test/src/util/constants.dart';
+import 'package:path/path.dart' as p;
+
+class HookFile {
+  HookFile._create({
+    required this.featureDir,
+    required this.package,
+    required this.fileName,
+    required this.import,
+  });
+
+  static HookFile create({
```

Comment:
There is a linter warning here. It might be fixed like that:
```suggestion
  factory HookFile.create({
```